examples/security-policy-all/versions.tf

```diff
@@ -18,11 +18,11 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = "~> 4.0"
+      version = ">= 4.0"
```

Comment:
For `/examples` you could also just remove the version line entirely, but this is also just fine.